### DISCUSSION TITLE
Expired URLs for Blues?

### DISCUSSION THREAD

**Discussion by theogaraj** [CMSgov/price-transparency-guide#742] [Q&A]
I've encountered this with a few Blues now.  The URLs in their index files have `Expires` tags, and often the expiration is in the past, resulting in an Access Denied error when attempting to download the in-network files.

How are others dealing with this?

**Comment by JR-CMS** [comment]
Are you experiencing an Access Denied error for a file within the month for which the file is required, or for a prior month?

**Reply by theogaraj** [reply]
@JR-CMS I'm getting Access Denied for a file within the month for which the file is required.

1. I download the index file from the payer's website.
2. Immediately after downloading the index file, I use the URLs within the index file to attempt to download the in-network files.
3. I get access denied errors for the in-network files.

I've so far had this happen with FL Blue, CareFirst and BCBS IL (those are the only Blues whose files I've been trying to get so far).  

It does not seem consistent, there have been some months where I have been successful in getting files from those payers.  If I were to guess, I'd say there's some problem of synchronization between the timings of when the rates files and index files are being published.

It would be great if there were a requirement for the payers to publish contact info of the person/dept responsible for these files.  We also work with the Exchange Qualified Health Plans for which payers are required to publish JSON data files as well as a responsible contact person, and for the most part we've had success with issue resolution there.

**Reply by theogaraj** [reply]
As I mentioned in my last message, behavior seems inconsistent.  Right now (4/30/2024 10:04am EST), CareFirst still exhibits the expired URL problem (I've detailed below how to reproduce) but BCBS IL and FL Blue files seem downloadable (whereas they were not on 4/22).

#### CareFirst
Index file: download the HMO file (`Table-Of-Content-CareFirst-HMO.json`) from https://individual.carefirst.com/individuals-families/mandates-policies/machine-readable-file.page
In-network: first in-network file URL in the index file is https://carefirstbcbs.mrf.bcbs.com/2024-02_690_08C0_in-network-rates_10_of_35.json.gz?&Expires=1711116039&Signature=3AvxqJ5WpfCjmmgcLsEWxXx14aLhe1zzqpWWUYLI8TdrCghMCuGSZ9kGtdMfUIIR9wNQSRDqsdawontEvaNnG1oh43cXwSlBly3sDKclzhGHw4YVAE7D~EYEWPEyU73iSQuPDLok7H62H5qIDpcUbywNCRVN-ScODxvqGiTDtNflvNrgfhcv8sQDfdag2FYeg6HjS4F-jlrahRGJxDRp0tOQwGA~iIR0NIPwzBeAuAXwi166qMJX0OtIznb6TNnMIy-gB0M6AleuTOa8rzWo2np5gUPNElHnEvR2d01hSSS6GlF6Yk5hUWvHdy0GgORJTHULMXcnCJbV~Ymd82ljsA__&Key-Pair-Id=K27TQMT39R1C8A
Attempting to download the file results in "Access denied" error.

**Reply by JR-CMS** [reply]
If you believe you are observing an instance of non-compliance with the Transparency in Coverage final rules, please submit it here: https://www.cms.gov/healthplan-price-transparency/contact-us.

Thank you

**Reply by theogaraj** [reply]
Ah very useful, thank you for this!

Is CMS considering requiring payers to publish contact/support information for these files?

**Comment by pbrumm** [comment]
I filed a ticket last week with cms around the carefirst issue.

PT Case Number: 44031162401042

**Reply by JR-CMS** [reply]
We received the submission, thank you.

**Reply by theogaraj** [reply]
@pbrumm thank you for reporting.  

If it's not too much to ask, I'd greatly appreciate an update here on this thread if you hear anything.

**Comment by pbrumm** [comment]
It looks like carefirst's data is fixed as of May 1.
